CMakeLists.txt

```diff
@@ -432,6 +435,13 @@ install(TARGETS ${S2_TARGETS}
         ARCHIVE DESTINATION "${CMAKE_INSTALL_LIBDIR}"
         LIBRARY DESTINATION "${CMAKE_INSTALL_LIBDIR}")
 
+install(TARGETS s2 EXPORT s2Targets)
```

Comment:
```suggestion
# create an export type "s2Targets" detailing the targetable artifacts created by this project
install(TARGETS s2 EXPORT s2Targets)
# install the export targets as a CMake config file in the share/s2 folder so that they can referenced 
# by downstream projects as `s2::s2` after a successful `find_package` call
```